.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
you need to add this file to ignore some files/folders, to not push them to git. For example node modules, as them should be installed by **yarn/npm** for every user that cloned repository.

Versioning and library updates controlled by **package.json** and **package-lock.json** files